### PR TITLE
Update Developer IntelliJ Setup.md

### PR DESCRIPTION
**Pull request by Guangyu-Joshua-Feng**
Reconfigure the IntelliJ Run configuration to locate the testing files.

### CODE REVIEW

**Review by edwardalee — COMMENTED** [review]
This looks good to me, but should we be steering users towards developing and running LF programs within IntelliJ at all?  Perhaps the example given here should not be running an LF program, but rather running some other gradle task?  E.g., if you want to debug the code generator in IntelliJ, you just want to invoke lfc, not run the generated code.

**Comment by lhstrh** [comment]
> This looks good to me, but should we be steering users towards developing and running LF programs within IntelliJ at all?

We should not, but this is also not user documentation but developer documentation. This setup is for people working on the compiler, who need access to a debugger, etc.

File: packages/documentation/copy/en/developer/Developer IntelliJ Setup.md [hunk 1]
```diff
@@ -50,7 +50,9 @@ You can set up a run configuration for running and debugging various Gradle task
 To set up a run configuration for the run task of lfc, right-click on "run" under org.lflang -> cli -> lfc -> Tasks -> application and click "Modify Run Configuration".
 This will create a custom run/debug configuration for you.
 
-In the Run/Debug Configurations dialog, click on the text box next to Tasks: and append args to specify the LF target. For example, `run --args 'test/Cpp/src/HelloWorld.lf'` Then click OK.
+In the Run/Debug Configurations dialog, click on the text box next to Tasks: use the full task name `cli:lfc:run` instead of just `run`.
```
Comment (lhstrh) [inline packages/documentation/copy/en/developer/Developer IntelliJ Setup.md:53]: I don't see "Tasks:" in the screenshot?

**Review by lhstrh — APPROVED** [review]
It tweaked this a bit -- I think it's ready to merge now.

**Pull request merged**